build/sbt-config/repositories-cn.template

```diff
@@ -0,0 +1,11 @@
+# As a Chinese developer facing network issues, you can accelerate the download speed of bootstrap/plugin/dependencies jar packages by executing the following command to configure the mirror:
+#
+# ```
+#   cp build/sbt-config/repositories-cn.template build/sbt-config/repositories-local
+# ```
+
+[repositories]
+  local
+  private: ${DEFAULT_ARTIFACT_REPOSITORY-file:///dev/null}
```

Comment:
If the user has configured it, include the `DEFAULT_ARTIFACT_REPOSITORY` in the list of repositories. cc @pan3793 

the variable substitution docs: https://www.scala-sbt.org/1.x/docs/Launcher-Configuration.html#Variable+Substitution

Reply:
the variable substitution required system property instead of the environment variable, updated in https://github.com/apache/incubator-celeborn/pull/1757/commits/48a1aa93801dedc3f2a4ed0b515efc24ae1e6662

Reply:
I see, thanks for fixing it.